OrbitGl/TimeGraph.cpp

```diff
@@ -189,6 +189,24 @@ void TimeGraph::ZoomTime(float a_ZoomValue, double a_MouseRatio) {
   SetMinMax(minTimeUs, maxTimeUs);
 }
 
+void TimeGraph::VerticalZoom(float zoom_value, double mouse_ratio) {
+  static double increment_ratio = 0.1;
+
+  // In this way, VerticalZoom is reversible
+  double scale = zoom_value > 0 ? 1 + increment_ratio : 1 / (1 + increment_ratio);
```

Comment:
The vertical zoom is inverted, usually we zoom in by scrolling the wheel towards the top of the mouse (see ctrl-scroll behavior in Visual Studio for example).

Reply:
Yes, you are right. Sorry for that.